files/homeassistant/hacs_version/individual_ams_lite.yaml

```diff
@@ -79,91 +82,96 @@ elements:
       action: none
     style:
       top: 77%
-      left: 45.7%
+      left: 12%
       text-align: center
       font-size: 1em
-      background-color: rgba(0,0,0,0.4)
-      box-shadow: 0 0 5px 5px rgba(0, 0, 0, 0.4)
+      background-color: var(--tray_2_bg)
+      padding: 7px
+      border: 2px solid rgba(255,255,255,0.6)
       border-radius: 50px
       pointer-events: none
       color: '#FFFFFF'
   - type: custom:config-template-card
     entities:
-      - sensor.{HA_AMS_DEVICE_NAME}_tray_4
+      - sensor.{HA_AMS_DEVICE_NAME}_tray_3
```

Comment:
Just want to validate that with your own AMS lite, the order now corresponds to the physical order on your unit when viewed from the same angle. I don't have an AMS lite so with the new image and angle shift, just wanted to confirm :)

Reply:
Good catch, i screwed the 3-4 positions.

This is the AMS lite ordering:
![image](https://github.com/WolfwithSword/Bambu-HomeAssistant-Flows/assets/38808827/c5bf35dc-1bbb-470f-98ab-89d70f3d316e)

I've fixed that (sorry, I have only pastel-colors from BL and can be difficult to see clearly 😅 ):
![image](https://github.com/WolfwithSword/Bambu-HomeAssistant-Flows/assets/38808827/8123d055-d6ee-49d7-8966-69434818bf68)

**PS:** what I meant with `Sorted AMS trays in the list` is that the `custom:config-template-card / state-label` were initially listed as 1-2-4-3, I just changed the order for the last two to be 1-2-3-4 (nothing critical, just listed as logical order)